.gitignore

```diff
@@ -6,8 +6,11 @@ docs/build/
 docs/site/
 benchmark/.results/*
 benchmark/.tune.jld
+benchmark/Manifest.toml
```

Comment:
I think this might be necessary for the benchmark CI to work properly? can you double check whether Graphs.jl is properly `dev`-ed into the environment without the manifest?

Reply:
The `dev` happens dynamically in https://github.com/tkf/BenchmarkCI.jl/blob/bca191f70d7fe54bb1c6ecc58e4c9b9d0ee8e4c6/src/BenchmarkCI.jl#L249

However I do believe the `] dev ..` should be hardwired (e.g. for local testing). But luckily we will have a way to do that in the future just with Project.toml because of https://github.com/JuliaLang/Pkg.jl/pull/3783

Reply:
Okay then let's keep it that way